content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -63,7 +63,6 @@ different Kubernetes components.
 | `AllowInsecureBackendProxy` | `true` | Beta | 1.17 | |
 | `AnyVolumeDataSource` | `false` | Alpha | 1.18 | 1.23 |
 | `AnyVolumeDataSource` | `true` | Beta | 1.24 | |
-| `AppArmor` | `true` | Beta | 1.4 | |
```

Comment:
In addition to removing this line for the first table, you should  add the following lines to the second table (for GA'ed feature):

```
| `AppArmor` | `true` | Beta | 1.4 | 1.25 |
| `AppArmor` | `true` | GA | 1.26 | - |
```

In the future, we will move the `AppArmor` lines to `content/en/docs/reference/command-line-tools-reference/removed-feature-gates.md` page when this gate is removed.

Reply:
Fixed in [b986c7f](https://github.com/kubernetes/website/pull/37191/commits/b986c7f1e75a04a4469276b4d1a55171fd57562f)